tests/test_vaccine_feed_ingest_schema_load.py

```diff
@@ -1,4 +1,5 @@
 import pydantic
+
```

Comment:
Please remove the unrelated changes from your PR (the three newlines and the change in the import in `vaccine_feed_ingest_schema/location.py`).

Unfortunately `isort` locally and `isort` in SuperLinter seem to run with different configs. We fixed this in `vaccine-feed-ingest` by updating how `isort` is run in CI, but have not yet made that change for this repo.

Reply:
OK, I'll update my changes to be super focused to just the `VaccineProviders` enum!